poll-billing.php

```diff
@@ -130,4 +139,14 @@ function CollectData($bill_id) {
     CollectData($argv[1]);
 }
 
+$poller_end  = microtime(true);
```

Comment:
This isn't important but if you have the time before this is merged. It would be good to add the $poller_time as a function rather than duplicate the code from poller.php (function can then be used there).